src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -239,7 +239,7 @@ const SearchFieldList = ({ onSearch }: Props) => {
       setDepartments(departments);
       setBuildings(buildings.map((b) => b.Description));
       setInvolvements(involvements);
-      setGender(gender);
+      setGender([...gender].sort());
```

Comment:
You could use [the `toSorted` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted) instead, which is easier to understand. It's semi-new to JavaScript, but has been supported in all major browsers since July 2023, and [has 91% support](https://caniuse.com/mdn-javascript_builtins_array_tosorted) across all users globally, so it should be safe to use.

```suggestion
      setGender(gender.toSorted());
```